### PR TITLE
Add safe buffer subclass which handles arrays

### LINKED ISSUE #2: Looping and outputting HTML?
I'm trying to do something like this, but running into syntax errors:

```
<div>
  {3.times.map do
    <p>Welcome to {"Rux" + " and"} Bridgetown!</p>
  end}
</div>
```

### PR DESCRIPTION
**Pull request by jaredcwhite**
This echoes similar work done to resolve https://github.com/camertron/rux/issues/2.

I haven't actually tested this yet with Rails…basically just copied over work I did to get the safe buffer working in Bridgetown. If you can think of a more elegant solution, let me know!

### CODE REVIEW

File: lib/rux-rails/safe_buffer.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'rux'
```
Comment (camertron) [inline lib/rux-rails/safe_buffer.rb:1]: Is this necessary?

**Review by camertron — CHANGES_REQUESTED** [review]
Huh so regular 'ol `ActiveSupport::SafeBuffer` doesn't handle arrays?

File: lib/rux-rails/safe_buffer.rb (hunk 1 above)
Comment (jaredcwhite) [reply lib/rux-rails/safe_buffer.rb:1]: Probably not…I noticed you had requires in some of the other files so I added it for consistency.

**Comment by jaredcwhite** [comment]
> Huh so regular 'ol `ActiveSupport::SafeBuffer` doesn't handle arrays?

It would appear not. Rux is handling things a bit differently than traditional ERB so I'm not surprised they're only thinking in terms of string expressions.

**Comment by camertron** [comment]
Published in v1.1.0 😄

**Pull request merged**